doc/doxygen/CMakeLists.txt

```diff
@@ -136,9 +125,24 @@ FOREACH (_source ${_source_files})
     SET (_last_year "${_year}")
   ENDIF()
 ENDFOREACH()
+SET(DEAL_II_PACKAGE_YEAR ${_last_year})
 MESSAGE(STATUS "doxygen output will have copyright date ${_last_year}")
 
 
+# Put the copyright year into the correct files
+CONFIGURE_FILE(
+  ${CMAKE_CURRENT_SOURCE_DIR}/scripts/mod_header.pl.in
+  ${CMAKE_CURRENT_BINARY_DIR}/scripts/mod_header.pl
+  )
+
+CONFIGURE_FILE(
+  ${CMAKE_CURRENT_SOURCE_DIR}/scripts/mod_footer.pl.in
+  ${CMAKE_CURRENT_BINARY_DIR}/scripts/mod_footer.pl
+  )
+
+
+
+
```

Comment:
What about we only use 2 or 3 newlines here? :smile:

Reply:
Sure, if that what you'd like :-)